### PR TITLE
(scribus) Migrated to AU

### PR DESCRIPTION
**Pull request by AdmiringWorm**

### CODE REVIEW

File: automatic/scribus/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  packageName    = 'scribus'
+  fileType       = 'exe'
+  file           = "$toolsPath\scribus-1.4.6-windows.exe"
+  file64         = "$toolsPath\scribus-1.4.6-windows-x64.exe"
+  softwareName   = ''
```
Comment (gep13) [inline automatic/scribus/tools/chocolateyInstall.ps1]: Is there something to put in here?  Or should it be removed?

Comment (AdmiringWorm) [inline automatic/scribus/tools/chocolateyInstall.ps1]: ops, yeah something is supposed to go there.
I must have just forgotten about it.
That's what I get for working on multiple packages at the same time. XD

**Comment by AdmiringWorm** [comment]
I'm updating this with the softwareName tomorrow, I'm more or less about to sign of for today.

File: automatic/scribus/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (gep13) [inline automatic/scribus/tools/chocolateyInstall.ps1]: @AdmiringWorm ha ha, not a problem 👍

**Comment by AdmiringWorm** [comment]
@gep13 just added the missing softwareName.
Just waiting for the appveyor build to run, then I'll merge this one.

**Comment by AdmiringWorm** [comment]
Damn, didn't notice that the archive became too large.
I'll need to rework it by downloading the files (or perhaps just to download the music backgrounds).
The archive became around 19 MB too large XD

**Pull request merged**